docusaurus/docs/learn/quickstarts/network/miniziti.bash

```diff
@@ -89,24 +95,37 @@ detectOs(){
     fi
 }
 
-getClientOttPath(){
-    local ID_PATH="/tmp/${MINIKUBE_PROFILE}-client.jwt"
-    if [[ $# -eq 1 ]]; then
-        case $1 in
-            macOS|Linux)     echo "$ID_PATH"
+makeMinizitiStateDir() {
+    case "$DETECTED_OS" in
+        MacOS)
+            state_dir="${XDG_STATE_DIR:-$HOME/Library/Application Support}/miniziti"
             ;;
-            Windows)         wslpath -w "$ID_PATH"
+        Linux|Windows)
+            state_dir="${XDG_STATE_DIR:-$HOME/.local/state}/miniziti"
             ;;
-        esac
-    else
-        logError "getClientOttPath() takes one param, DETECTED_OS"
-        return 1
+        *)
+            logError "Unknown os: $DETECTED_OS"
+            exit 1
+    esac
+
+    if [[ ! -d "$state_dir" ]]; then
+        logDebug "Creating miniziti state directory: $state_dir"
+        mkdir -p "$state_dir"
     fi
 
+    echo "$state_dir"
+}
+
+pathToNative() {
+    local path="$1"
+    case "$DETECTED_OS" in
+        Windows) wslpath -w "$path" ;;
```

Comment:
```suggestion
        Windows) wslpath -w "$path" | sed -E 's|\\|\\\\|g' ;; # escape backslashes in UNC path from interpretation by logger()
```

Reply:
Can you try again with the new changes?

Reply:
I'm getting this consistently with https://raw.githubusercontent.com/openziti/ziti-doc/feature/multi_profile_support/docusaurus/docs/learn/quickstarts/network/miniziti.bash

```bash
INFO cluster dns test succeeded
+ logDebug 'fetching admin password from k8s secret to log in to ziti mgmt'
+ logger 'fetching admin password from k8s secret to log in to ziti mgmt'
+ local caller=logDebug
+ ((  1 < 1  ))
+ local 'message=fetching admin password from k8s secret to log in to ziti mgmt'
+ [[ fetching admin password from k8s secret to log in to ziti mgmt =~ ^r'(.+)'$ ]]
+ caller_level=Debug
+ ((  MINIZITI_DEBUG  ))
+ line='DEBUG fetching admin password from k8s secret to log in to ziti mgmt'
+ [[ -n '' ]]
+ echo -e 'DEBUG fetching admin password from k8s secret to log in to ziti mgmt'
+ getAdminSecret
+ kubectl_wrapper get secrets ziti-controller-admin-secret --namespace miniziti --output 'go-template={{index .data "admin-password" | base64decode }}'
+ xargs ziti edge login miniziti-controller.miniziti.ziti:443 --yes --cli-identity miniziti --username admin --password
+ minikube kubectl --profile miniziti -- --context miniziti get secrets ziti-controller-admin-secret --namespace miniziti --output 'go-template={{index .data "admin-password" | base64decode }}'
The connection to the server miniziti-controller.miniziti.ziti:443 was refused - did you specify the right host or port?
```

Reply:
This should also be resolved.